version/version.go

```diff
@@ -13,7 +13,7 @@ const (
 )
 
 func Agent() string {
-	return fmt.Sprintf("pactus/%s", Version())
+	return fmt.Sprintf("node=pactus/version=%s/os=%s/architecture=%s", Version(), runtime.GOOS, runtime.GOARCH)
```

Comment:
We need another key=value about peer is from GUI or Daemon

Reply:
@Ja7ad I checked the code, there are some places we are calling the Agent function but we don't have access to find out which client is calling the function.

should we pass it from other modules? (for example, adding a ClientType field to www modules, so in response we can use it to pass to the Aget)